package.json

```diff
@@ -43,8 +43,9 @@
     "@nx/rollup": "16.9.1",
     "@nx/web": "16.9.1",
     "@nx/workspace": "16.9.1",
-    "@openfeature/server-sdk": "^1.6.2",
     "@openfeature/core": "^0.0.16",
+    "@openfeature/flagd-core": ">=0.1.1",
```

Comment:
@toddbaert I had to add this as a dev dependency. This is needed for e2e testing.

Reply:
You should be able to remove this by updating to more recent versions of server or client (1.7.2 and 0.4.5), respectively.

Reply:
I think I confused you. This here refers to shared module we introduced for flagd: `shared/flagd-core` [^1]

e2e tests fail as it rely on root level package.json. See this CI failure which prompted me to introduce this [^2]

[^1]: https://github.com/open-feature/js-sdk-contrib/tree/main/libs/shared/flagd-core
[^2]: https://github.com/open-feature/js-sdk-contrib/actions/runs/6750802408/job/18353826560#step:7:78

Reply:
I misread this as the shared SDK core module. You are right this is needed here.